log.go

```diff
@@ -59,7 +62,24 @@ func (l *logger) submit(logs []log) {
 	finalLogs := bytes.NewBuffer(nil)
 	enc := json.NewEncoder(finalLogs)
 	enc.Encode(logBatch{logs})
-	l.client.Post(l.endpoint.String(), "application/json", finalLogs)
+
+	req, err := http.NewRequest(http.MethodPost, l.endpoint.String(), finalLogs)
+	if err != nil {
+		goLogger.Errorw("failed to create http request to submit saturn logs", "err", err)
+		return
+	}
+
+	req.Header.Set("Content-Type", "application/json")
+	req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", l.jwt))
+
+	resp, err := l.client.Do(req)
```

Comment:
rather than having this a specific `saturn logging needs a jwt bearer token` - which is a pretty different level of abstraction than the rest of this library so far - i wonder if we can have this header set by the configured http client.

so e.g. we set the specific client/url at https://github.com/ipfs/bifrost-gateway/blob/main/blockstore.go#L111
we could do something like the final code block at https://developer20.com/add-header-to-every-request-in-go/ to configure the JWT token in the same place, which seems more local than splitting it between there and here.

Reply:
I have removed this token from the config.